phones/tests/models_tests.py

```diff
@@ -53,7 +53,9 @@ def make_phone_test_user():
 
 
 def upgrade_test_user_to_phone(user):
-    random_sub = random.choice(settings.SUBSCRIPTIONS_WITH_PHONE.split(","))
+    random_sub = random.choice(
+        list(filter(None, settings.SUBSCRIPTIONS_WITH_PHONE.split(",")))
+    )
```

Comment:
This is some lisp wizard stuff 🧙‍♂️ . It works, but I'll continue to use my version:

```suggestion
    random_sub = random.choice(
        [s for s in settings.SUBSCRIPTIONS_WITH_PHONE.split(",") if s])
    )
```